Altis_Life.Altis/config/Config_Clothing.hpp

```diff
@@ -20,47 +20,58 @@ class Clothing {
         conditions = "";
         side = "civ";
         uniforms[] = {
-            { "NONE", "Remove Uniform", 0, "" },
-            { "U_C_Poloshirt_stripped", "Poloshirt stripped", 125, "" },
-            { "U_C_Poloshirt_redwhite", "Poloshirt Red/White", 150, "" },
-            { "U_C_Poloshirt_salmon", "Poloshirt Salmon", 175, "" },
-            { "U_C_Poloshirt_blue", "Poloshirt Blue", 250, "" },
-            { "U_C_Poor_2", "Rag tagged clothes", 250, "" },
-            { "U_C_Poloshirt_burgundy", "Poloshirt Burgundy", 275, "" },
-            { "U_C_Poloshirt_tricolour", "Poloshirt Tricolor", 350, "" },
-            { "U_IG_Guerilla2_2", "Green stripped shirt & Pants", 650, "" },
-            { "U_IG_Guerilla3_1", "Brown Jacket & Pants", 735, "" },
-            { "U_OrestesBody", "Surfing On Land", 1100, "" },
-            { "U_IG_Guerilla2_3", "The Outback Rangler", 1200, "" },
-            { "U_C_HunterBody_grn", "The Hunters Look", 1500, "" },
-            { "U_C_WorkerCoveralls", "Mechanic Coveralls", 2500, "" },
-            { "U_NikosAgedBody", "Casual Wears", 5000, "" }
+            { "NONE", $STR_C_Remove_uniforms, 0, "" },
+            { "U_C_Poloshirt_stripped", "", 125, "" },
+            { "U_C_Poloshirt_redwhite", "", 150, "" },
+            { "U_C_Poloshirt_salmon", "", 175, "" },
+            { "U_C_Poloshirt_blue", "", 250, "" },
+            { "U_C_Poor_2", "", 250, "" },
```

Comment:
Line 28 can be moved two lines down so that way the six [commoner clothes](https://community.bistudio.com/wiki/Arma_3_CfgWeapons_Equipment#U_C_Poloshirt_stripped) are in order.